README.md

```diff
@@ -0,0 +1,19 @@
+# Mailroom app
+
+- This app allows someone to add donors to a database and generate a thank you letter.  It also allows the user to view the list of donors in a table format, sorted by total amount donated.  The data is persisted between application runs.
+
+# Instructions
+Clone this repo, start create new virtual env, install required plugins using:
+```
+pip install -r requirements.pip
+```
+
+Usage
+```
+python3 src/mailroom.py
```

Comment:
might be nice here to get some description of the stages of the app, and what the user might expect to see.